docs/src/04-background-color.stories.mdx

```diff
@@ -1,22 +1,31 @@
-import { Meta } from '@storybook/addon-docs/blocks';
-import { TokenSection } from './lib/TokenSection';
-import { background } from '@wmde/wikit-tokens';
+import { Meta } from "@storybook/addon-docs/blocks";
+import { TokenSection } from "./lib/TokenSection";
+import { background } from "@wmde/wikit-tokens";
 
 <Meta title="Design Tokens/Background color" />
 
 ## Background color
 
 ### Base
+
 <TokenSection tokens={background.color.base} />
 
+### Field
```

Comment:
Mental note: I don't think these are proper alias tokens. A general restructuring and renaming of Alias tokens is a must. To whoever is reading this: no worries, components won't be affected 😎